src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -30,7 +30,7 @@ private string NuGetCachePath
             }
         }
 
-        private string SentinelPath => Path.Combine(NuGetCachePath, SENTINEL);
+        private string _sentinelPath => Path.Combine(NuGetCachePath, SENTINEL);
```

Comment:
Hmm sure, but we have to pass either a nugetcachesentinel object or some other object into the firsttimeconfigure method